devtools/server/actors/replay/module.js

```diff
@@ -952,6 +953,17 @@ function Debugger_getPossibleBreakpoints({ sourceId, begin, end }) {
   }
 }
 
+function Target_getPossibleBreakpointsForMultipleSources({ sourceIds }) {
+  return { possibleBreakpoints:
+    sourceIds.map((sourceId => {
```

Comment:
`possibleBreakpoints:` should go on this line (we don't have eslint running for gecko-dev, alas).